### PR TITLE
fix(VAutocomplete): Disable browser autocomplete, fix#8395

### PR DESCRIPTION
**Pull request by simjes**
## Description
Disable browser autocomplete by default in VAutocomplete

## Motivation and Context
Component should suggest autocompletion items, not the browser. 

## How Has This Been Tested?
Tested it in the docs by recreating the example that is linked in #8395

## Markup:

<details>

```vue
<template>
  <v-app>
    <v-content>
      <v-container>
        <v-form ref="form" @submit.prevent="submit()">
          <v-autocomplete v-model="item" :items="items" placeholder="Type something..." id="item"></v-autocomplete>

          <v-btn type="submit" color="primary" :disabled="isValid">Submit</v-btn>
        </v-form>
      </v-container>
    </v-content>
  </v-app>
</template>

<script>
export default {
  name: "App",
  data() {
    return {
      items: ["One", "Two", "Three"],
      item: ""
    };
  },

  computed: {
    isValid() {
      return !this.item;
    }
  },

  methods: {
    submit() {
      alert(`Item ${this.item}`);
      this.$refs.form.reset();

    }
  }
};
</script>
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.

### CODE REVIEW

File: packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts [hunk 1]
```diff
@@ -281,7 +281,10 @@ export default VSelect.extend({
       const input = VTextField.options.methods.genInput.call(this)
 
       input.data = input.data || {}
-      input.data.attrs = input.data.attrs || {}
+      input.data.attrs = {
```
Comment (MajesticPotatoe) [inline packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts]:
```
input.data.attrs = input.data.attrs || {}
input.data.attrs.autocomplete = 'off'
```

Comment (simjes) [reply packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts]: I just assumed that you should be able to turn browser autocomplete on, but that doesn't make sense at all :D Will update

Comment (MajesticPotatoe) [reply packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts]: autocomplete is just passed through to the native input, the `input.data.attrs || {}` is to safeguard against a null or empty value. would likely throw an error if you spread a null value.

**Comment by vcastro45** [comment]
This issue has been fixed by v2.1.2 but v2.1.3 is has regression.

In v2.1.3, the autocomplete attribute is set to `"disabled"` by default but it is not a valid value.
The doc for the `<input>` HTML tag says the values can be `"on"` or `"off"`: 

https://www.w3schools.com/tags/att_input_autocomplete.asp

**Comment by MajesticPotatoe** [comment]
@vcastro45 `disabled` is in fact not a valid value, however it was chosen to combat against chrome ignoring the value of `off` in certain scenarios while still effectively disabling it for other browsers. This of course is overridden if you set the value yourself maintaining functionality for people who have explicitly set it. If you are experiencing that this is not the case please open a new issue.

**Comment by vcastro45** [comment]
@MajesticPotatoe In what kind of scenario Chrome doesn't disable the input ?
I currently do my tests using Chrome and since I updated vuetify from 2.1.2 to 2.1.3, the browser autocomplete is shown everywhere...

**Comment by MajesticPotatoe** [comment]
> If you are experiencing that this is not the case please open a new issue.
https://issues.vuetifyjs.com/

**Pull request merged**